pyanaconda/modules/timezone/timezone_interface.py

```diff
@@ -79,6 +79,15 @@ def SetTimezoneWithPriority(self, timezone: Str, priority: UInt16):
         """
         self.implementation.set_timezone_with_priority(timezone, priority)
 
+    def GetTimezones(self) -> Dict[Str, List[Str]]:
+        """Get valid timezones.
+
+        Return a dictionary, where keys are region ids and values are lists of timezone names in the region.
+
+        :return: a list of timezones
```

Comment:
The return type is wrong now, because you have changed the proposed API. Pinging @VladimirSlavik to look at it.

Reply:
IMO this isn't a big problem, actually. The data needs to come from somewhere anyway, in some form. If we already collate this in the backend, it might be advantageous to just use it.

It's true the original suggestion is more UI agnostic, though...